register-modules.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Ok, fixing file permissions. As for running locally, The tests are all I got, but that is unit testing. I then use the test AWS account and deploy it there when testing everything together.